examples/test_ibm_cloud_shell_v1_examples.py

```diff
@@ -0,0 +1,156 @@
+# -*- coding: utf-8 -*-
+# (C) Copyright IBM Corp. 2021.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Examples for IbmCloudShellV1
+"""
+
+from ibm_cloud_sdk_core import ApiException, read_external_sources
+import os
+import pytest
+from ibm_platform_services.ibm_cloud_shell_v1 import *
+
+#
+# This file provides an example of how to use the IBM Cloud Shell service.
+#
+# The following configuration properties are assumed to be defined:
+# IBM_CLOUD_SHELL_URL=<service base url>
+# IBM_CLOUD_SHELL_AUTH_TYPE=iam
+# IBM_CLOUD_SHELL_APIKEY=<IAM apikey>
+# IBM_CLOUD_SHELL_AUTH_URL=<IAM token service base URL - omit this if using the production environment>
+#
```

Comment:
Just a reminder about that ACCOUNT_ID should come here too

Reply:
Done